project3/styles.css

```diff
@@ -0,0 +1,321 @@
+    background-color: #163346;
+}
+
+form {
+    background-color: #1e2019;
+    max-width: 70%;
+    margin-right: auto;
+    margin-left: auto;
+    padding: 20px;
+
+    display: block;
+    position: relative;
+    bottom: 50px;
+}
+
+label {
+    font-family: "Benchnine", monospace;
+    font-size: 22px;
+    color: #cececd;
+}
+
+
+legend {
+    font-family: "Benchnine", monospace;
+    font-size: 22px;
+    color: #cececd;
+}
+
+.sr-only {
+    border: 0;
+    clip: rect(0 0 0 0);
+    overflow: hidden;
+    margin: -1px;
+    padding: 0;
+    position: absolute !important;
+    height: 1px;
+    width: 1px;
+}
+
+section {
+    background-color: #1e2019;
+    margin-right: auto;
+    margin-left: auto;
+    max-width: 70%;
+}
+
+.input,
+.textarea,
+.select {
+    font-family: inherit;
+    font-size: inherit;
+}
+
+
+.age {
+    padding-bottom: 30px;
+}
+
+fieldset {
+    border: 0px;
+    padding: 0px;
+    margin-right: auto;
+    margin-left: auto;
+    max-width: 70%;
+    display: inline-block;
+}
+
+select {
+    cursor: pointer;
+    width: 100%;
+    height: 70px;
+    font-family: "Benchnine", monospace;
+    font-size: 18px;
+    text-align: center;
+    color: #1e2019;
+
+}
+
+span {
+    display: block;
+}
+
+.shield {
+    width: 240px;
+    height: 239px;
+    display: block;
+    position: relative;
+    top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    background-color: #cececd;
+}
+
+.icons {
+    padding: 10px;
+    text-align: center;
+}
+
+@media (min-width:900px) {
+    .icons {
+        display: inline-block;
+        width: 30%;
+    }
+
+    select {
+        cursor: pointer;
+        width: 80%;
+        height: 70px;
+        font-family: "Benchnine", monospace;
+        font-size: 18px;
+        text-align: center;
+        color: #1e2019;
+
+    }
+}
+
+@media (min-width:400px) {
+
+    select {
+        cursor: pointer;
+        width: 80%;
+        height: 70px;
+        font-family: "Benchnine", monospace;
+        font-size: 18px;
+        text-align: center;
+        color: #1e2019;
+
+    }
+}
+
+.bulb {
+    width: 100px;
+    height: 100px;
+    padding-bottom: 40px;
+    padding-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.clock {
+    width: 100px;
+    height: 100px;
+    padding-bottom: 40px;
+    padding-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+
+}
+
+.iconsubs {
+    padding-top: 25px;
+}
+
+.muscle {
+    width: 100px;
+    height: 100px;
+    padding-bottom: 40px;
+    padding-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+textarea {
+    height: 120%;
+    width: 75%;
+}
+
+textarea:hover {
+    border: 1px solid #71f79f;
+}
+
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-animation: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 1px solid #cececd;
+    height: 20px;
+    width: 20px;
+}
+
+input[type="radio"] {
+    border-radius: 50%;
+}
+
+
+input[type="checkbox"]:checked {
```

Comment:
Consider combining the styles for the checked checkbox and radio inputs since they are the same.